src/elexmodel/client.py

```diff
@@ -169,8 +170,33 @@ def get_aggregate_list(self, office, aggregate):
         raw_aggregate_list = base_aggregate + [aggregate]
         return sorted(list(set(raw_aggregate_list)), key=lambda x: AGGREGATE_ORDER.index(x))
 
-    def get_national_summary_votes_estimates(self, nat_sum_data_dict=None, called_states={}, base_to_add=0, alpha=0.99):
-        return self.model.get_national_summary_estimates(nat_sum_data_dict, called_states, base_to_add, alpha)
+    def get_national_summary_votes_estimates(
+        self, nat_sum_data_dict=None, called_states=None, base_to_add=0, alpha=0.99
+    ):
+        if self.model is None:
+            raise ModelClientException(
+                "Must call the get_estimands() method before get_national_summary_votes_estimates()."
+            )
+
+        if "nat_sum_data" in self.results_handler.final_results:
```

Comment:
After calling `get_national_summary_votes_estimates` once this would always return the same national summary estimates, rather than creating new ones with the new model updates.

Reply:
That's right.  The current way this operates and the code is written, there is no `self.model` without calling `get_estimates()`.  That flow is maintained here.  There's no reason to re-compute the national summary estimates without calling `get_estimates()` first, so might as well return the ones using the model from the previous `get_estimates()` call. 🤷🏻‍♀️

Reply:
I mean, unless there is a reason to re-compute them, in which case I'll take this out lol 🤔

Reply:
The national summary prediction might change any time the model is run